src/test/java/com/google/api/codegen/testsrc/common/samples/delete_shelf.sample.yaml

```diff
@@ -0,0 +1,53 @@
+type: com.google.api.codegen.samplegen.SampleConfigProto
+config_schema_version: 1.2.0
+samples:
+  # Behavior change:
+  # old config: if region_tag is unspecified or region_tag is not unique, derive the 
```

Comment:
nit: consider being more explicit in case these comments stay around longer than intended:
```
# Behavior change when switching from gapic configs to sample configs:
# gapic config: ...
# 
# sample config: ...
```

In the other configs as well.